src/panel/poi_panel.js

```diff
@@ -203,6 +204,14 @@ PoiPanel.prototype.emptyClickOnMap = function() {
   }
 }
 
+PoiPanel.prototype.goToFeedbackUrl = function() {
```

Comment:
This function is exactly the same as the one in `category_panel.js`. Couldn't you factorize it instead of redeclaring it?

Reply:
Moved this func in the Poi class 👍